js/viz/core/renderers/renderer.js

```diff
@@ -1946,19 +1947,38 @@ Renderer.prototype = {
         return elem.attr({ text: text, x: x || 0, y: y || 0 });
     },
 
-    linearGradient: function(stops) {
-        const id = getNextDefsSvgId();
-        const that = this;
-        const gradient = that._createElement('linearGradient', { id: id }).append(that._defs);
+    linearGradient: function(stops, _id, rotationAngle) {
```

Comment:
can we write like

_linearGradient: function(stops, id = getNextDefsSvgId(), rotationAngle) {_

?

Reply:
yes, we can